x-pack/plugins/observability/server/services/slo/transform_generators/apm_transaction_duration.ts

```diff
@@ -17,7 +17,7 @@ import { getSLOTransformTemplate } from '../../../assets/transform_templates/slo
 import { SLO, APMTransactionDurationIndicator } from '../../../domain/models';
 import { TransformGenerator } from '.';
 
-const APM_SOURCE_INDEX = 'metrics-apm*';
+const DEFAULT_APM_INDEX = 'metrics-apm*';
```

Comment:
Any reason to define `DEFAULT_APM_INDEX` twice? (also in [apm_transaction_error_rate.ts](https://github.com/elastic/kibana/pull/147281/files#diff-21eb1f91184393609c9483ec0f5c84d8cd0ee5ffb934065f338539bb79e2bad8))